e2e-tests/src/utils.rs

```diff
@@ -30,15 +38,26 @@ pub fn accounts(seeds: Option<Vec<String>>) -> Vec<KeyPair> {
     seeds.into_iter().map(keypair_from_string).collect()
 }
 
+pub fn get_first_account(accounts: &[KeyPair]) -> KeyPair {
+    accounts.get(0).expect("No accounts passed").to_owned()
+}
+
 pub fn get_first_two_accounts(accounts: &[KeyPair]) -> (KeyPair, KeyPair) {
```

Comment:
These two functions here are kinda bizarre – you can just use `[0]` and `[1]` everywhere and this will panic when out of bounds (which is fine, the test will fail, and it's terribly unexpected if I understand correctly – it would mean the test has been set up incorrectly, right?).

Reply:
done